### PR TITLE
Merge feature/o11y packs

### PR DESCRIPTION
**Pull request by nr-kkenney**
## Summary
Merge in all changes from the feature branch into develop.

Resolves: #1606

### CODE REVIEW

File: src/data/constants.js [hunk 1]
```diff
@@ -10,3 +10,52 @@ export const SPLIT_TRACKING_EVENTS = {
 
 export const SDK_BASE_URL =
   'https://d1zobbh8kytrtv.cloudfront.net/platform/doc-app';
+
+// FIXME: update this to production URL when deployed / launched
+export const NR1_LOGIN_URL = 'https://staging-login.newrelic.com/login';
```
Comment (aswanson-nr) [inline src/data/constants.js:15]: Needs to be updated

File: src/data/constants.js [hunk 2]
```diff
@@ -10,3 +10,52 @@ export const SPLIT_TRACKING_EVENTS = {
...
+export const QUICKSTART_ALERT_TYPES = {
+  BASELINE: 'BASELINE',
+  OUTLIER: 'OUTLIER',
+  STATIC: 'STATIC',
+};
+
+export const SHIELD_LEVELS = ['NEWRELIC', 'VERIFIED'];
+
+export const QUICKSTARTS_REPO =
+  'https://github.com/newrelic/newrelic-observability-packs';
```
Comment (aswanson-nr) [inline src/data/constants.js:56]: Needs to be updated

File: src/utils/get-pack-nr1-url.js [hunk 3]
```diff
@@ -0,0 +1,60 @@
+// FIXME: update this to production URL when deployed / launched
+const NR1_BASE_URL = 'https://staging-one.newrelic.com';
+const NR1_BASE_URL_LOCAL = 'https://dev-one.newrelic.com';
```
Comment (aswanson-nr) [inline src/utils/get-pack-nr1-url.js]: Needs to be updated

File: src/pages/students.js [hunk 4]
```diff
@@ -199,7 +199,7 @@ const StudentsPage = ({ location }) => {
               <p className={styles.question}>
                 How do I access New Relic Student Edition?
               </p>
-              <div className={styles.answer}>
```
Comment (aswanson-nr) [inline src/pages/students.js:202]: I don't think we should be seeing changes to this file

File: src/templates/ApiReferenceTemplate.js [hunk 5]
```diff
@@ -37,7 +37,7 @@ const ApiReferenceTemplate = ({ data, location }) => {
         <PageLayout.Content>
           {description && (
             <Section className="intro-text">
-              <Markdown source={description} />
```
Comment (aswanson-nr) [inline src/templates/ApiReferenceTemplate.js:40]: I don't think we want this change

File: src/templates/ComponentReferenceTemplate.js [hunk 6]
```diff
@@ -61,7 +61,7 @@ const ComponentReferenceTemplate = ({ data, location }) => {
         <PageLayout.Header title={name} />
         <PageLayout.Content>
           <Section className="intro-text">
-            <Markdown source={componentDescription} />
```
Comment (aswanson-nr) [inline src/templates/ComponentReferenceTemplate.js:64]: I don't think we want this change

File: src/utils/get-pack-nr1-url.js (hunk 3 above)
Comment (LizBaker) [reply src/utils/get-pack-nr1-url.js]: this just got merged in 🎉

**Review by LizBaker — APPROVED** [review]

**Pull request merged**